apps/src/code-studio/assets/searchAssets.js

```diff
@@ -23,7 +23,7 @@ export function searchAssets(
   // Example: searchQuery "bar"
   //   Will match: "barbell", "foo-bar", "foo_bar" or "foo bar"
   //   Will not match: "foobar", "ubar"
-  const searchRegExp = new RegExp('(?:\\b|_)' + searchQuery, 'i');
+  const searchRegExp = new RegExp('(?:\\s+|_|^)' + searchQuery, 'i');
```

Comment:
`\b` [doesn't work with unicode](https://medium.com/@shiba1014/regex-word-boundaries-with-unicode-207794f6e7ed) so I replaced it with spaces or a word beginning. Let me know if you think I missed anything!

Reply:
we should probably also include `-`